actions/project.js

```diff
@@ -21,6 +21,13 @@ const {
     getAioLogger, getUrlInfo, handleExtension, getFloodgateUrl, getDocPathFromUrl
 } = require('./utils');
 
+const PROJECT_STATUS = {
+    NOT_STARTED: 'NOT STARTED',
+    COMPLETED: 'COMPLETED',
+    COMPLETED_WITH_ERROR: 'COMPLETED WITH ERROR',
```

Comment:
@sirivuri I did not realize that `COMPLETED_WITH_ERROR` status was used for failures. This status is intended to be used when the copy or promote action is completed but perhaps some documents failed to copy. You should add one more `FAILED` status and set that instead.